pkg/cmd/subcommand/install.go

```diff
@@ -211,14 +216,18 @@ func (i *Install) RunInstall(cl *k8s.Clientset, cmd *cobra.Command) error {
 	operator.Inventory = inventoryPending
 	inventoryExist := getExistComponentsInventory(ctx, cl)
 
-	fmt.Fprintln(w, ti.BeforeTask("Start installing OpenFunction and its dependencies.\n"+
-		"Here are the components and corresponding versions to be installed:"))
+	util.BeforeTask("Start installing OpenFunction and its dependencies.\n" +
+		"Here are the components and corresponding versions to be installed:")
 	printInventory(inventory.GetVersionMap(inventoryPending))
-	if !reflect.DeepEqual(inventoryExist, map[string]bool{}) {
-		fmt.Fprintln(w, ti.BeforeTask("The following components already exist:"))
+	if !reflect.DeepEqual(inventoryExist, map[string]bool{}) && !i.Force {
+		if !i.Upgrade {
+			util.BeforeTask("The following components already exist (will be skipped):")
```

Comment:
The following components already exist (will be skipped):
=>
The following existing components will be skipped: